lib/tasks/assets.rake

```diff
@@ -78,5 +78,11 @@ namespace :assets do
   end
 
   desc 'Export frontend locale files'
-  task export_locales: ['i18n:js:export']
+  task :export_locales do
+    puts "Exporting I18n.js locales"
+    time = Benchmark.realtime do
+      I18nJS.call(config_file: Rails.root.join('config/i18n.yml'))
+    end
+    puts "=> Done in #{time.round(2)}s"
```

Comment:
I would have expected this to be called somewhere for both the test as well as the production environment. How is this dealt with?

Reply:
This bit is actually unchanged. The rake task `assets:precompile` is extended to call this task, and just its contents have changed now.

Here's the enhance call: https://github.com/opf/openproject/blob/a602295a6a7f8d763f21756b5ee8552f35065e42/lib/tasks/assets.rake#L33-L35

Reply:
Ah, I didn't notice that the `export_locales` existed before and thus was already referenced in the build pipeline. Thanks for pointing it out.